docs/user-manual/consume.md

```diff
@@ -5,25 +5,73 @@ slug: /consume
 ---
 
 # Consume fact statements
+
 A fact statement is a datum that the Orcfax network publishes to a Cardano
 blockchain transaction. This data point becomes an input to Cardano smart
 contracts and dApps.
 
-All fact statements are structured as [JSON-LD](https://json-ld.org/) files
+## UTxO model
+
+A single fact statement is associated with a single [UTxO][utxo-1]. Each UTxO
+has a [reference input][reference-1] encoded as an [inline datum][datum-1].
+
+Reference inputs are published using a COOP authentication token with a COOP
+minting policy identifier, currently:
+
+* preprod: `5ec8416ecd8af5fe338068b2aee00a028dc1f4c0cd5978fb86d7c038?`
+* mainnet: `84da51fd8041b6a1073f1154ddde0c846d71c4081fc1f7ba66e0e6b5?`
+
+This is important as fact statements should be verified as coming from a known
+source and this also helps to filter all of the UTxO at a given smart contract
+address where anyone has the ability to send other UTxO.
+
+Users will therefore need to use the following information to identify the
+latest facts.
+
+* Minting policy ID.
+* Datum format, including:
+  * Most recent (`ValueReference -> PropertyValue[1] -> value)`, i.e. the
+  largest POSIX timestamp compared to other fact statement datum,
+  * Feed name, e.g. "ADA-USD".
+
+> Note: Users on preprod will, for example, find a volume of test data that
+> hasn't been retired that needs to be ignored using the latest preprod policy
+> ID and fact schema.
+
+Unconsumed reference inputs are those still available for smart contracts to use
+on-chain. Spending a UTxO with a reference input renders it unusable in a smart
+contract (though the data will be visible in the historical transaction).
+Orcfax's mainnet V1 will operate a policy of two unspent fact UTxO remaining
+on-chain at all times. Consuming reference inputs will sometimes be referred to
+as garbage collection or *retiring* of fact statements.
+
+For more information about the use of reference input and inline datum in COOP
+see COOP's [design document][coop-design-1].
+
+[coop-design-1]: https://github.com/mlabs-haskell/cardano-open-oracle-protocol/blob/9e9c9aedba84d32e424b1dd116b4734e1a42f3bc/coop-docs/00-design.md#cardano-features-enabling-oracles
+
+[utxo-1]: https://docs.cardano.org/learn/eutxo-explainer/
+[reference-1]: https://docs.cardano.org/cardano-testnet/about/feature-overview/#referenceinputs(cip-31)
+[datum-1]: https://docs.cardano.org/cardano-testnet/about/feature-overview/#inlinedatums(cip-32)
+
+## Datum structure
+
+All fact statements are structured as [JSON-LD](https://json-ld.org/) objects
 before they are serialized into the Concise Binary Object Representation
 ([CBOR](https://cbor.io/)) that is used in Cardano transactions.
 
 ![Fact Statement](/img/2023-09-30--Orcfax--fact-statement.jpg)
```

Comment:
link did not work for me

Reply:
Just this one? 

* /img/2023-09-30--Orcfax--fact-statement.jpg

This will render when compiled but otherwise does look broken on GitHub